index.html

```diff
@@ -20,10 +20,16 @@
       <footer>
         <!-- {% include banner.html %}  -->
         <div class="bounds">
-          <section class="hero-footer__section">
+          <section class="hero-footer__section" itemscope itemtype="http://schema.org/SoftwareApplication" >
+              <meta itemprop="ApplicationCategory"  content="hybrid mobile applications, web development"/>
```

Comment:
Fix the indent of the meta tags - should just be 2 spaces more than the `<section` tag above it.